src/app/store/persistor.ts

```diff
@@ -0,0 +1,9 @@
+import { EnhancedStore } from '@reduxjs/toolkit';
+import { persistStore } from 'redux-persist';
+
+import { ApplicationState } from '.';
+
+export const createPersistor = (store: EnhancedStore<ApplicationState>) =>
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```

Comment:
Why?

Reply:
For some reason the `redux-persist` typing doesn't include `manualPersist`